IS7024/Pages/LudlowGarage.cshtml

```diff
@@ -6,72 +6,63 @@
-    {
+                foreach (QuickType.Event eventss in events)
+                {
 
-        string date = eventss.Start.Date.ToString();
-        string[] dars = date.Split('/');
-        string day = dars[0];
-        string month = dars[1];
-        string year = dars[2];
-        string[] year1 = year.Split(' ');
-        string yearclean = year1[0];
+                    string date = eventss.Start.Date.ToString();
+                    string[] dateSplit = date.Split('/');
+                    string day = dateSplit[0];
+                    string month = dateSplit[1];
+                    string year = dateSplit[2];
+                    string[] year1 = year.Split(' ');
+                    string yearclean = year1[0];
 
 
-        if (eventss.Start.Datetime != null)
+                    if (eventss.Start.Datetime != null)
 
-        {
-            string[] tars = eventss.Start.Datetime.Split('T');
-            string[] tars2 = tars[1].Split(':');
-            string mhour = tars2[0];
-            string mmin = tars2[1];
+                    {
+                        string[] dateTimeSplit = eventss.Start.Datetime.Split('T');
+                        string[] timeSplit = dateTimeSplit[1].Split(':');
+                        string mhour = timeSplit[0];
+                        string mmin = timeSplit[1];
 
-            var newhour = Convert.ToInt32(mhour);
-            int realhour = newhour - 12;
-            string starttime = realhour + ":" + mmin + " PM";
+                        var newhour = Convert.ToInt32(mhour);
+                        int realhour = newhour - 12;
+                        string starttime = realhour + ":" + mmin + " PM";
 
 
-                     
-            {
-               
 
-                    <tr>
-                        <td>@day/@month/@yearclean</td>
-                        <td>@eventss.Performance[0].Artist.DisplayName</td>
-                        <td>@eventss.Status</td>
-                        <td>@starttime</td>
-                        <td>@sun</td>
-                        <td><a href="@eventss.Uri" target="_blank">Get Tickets</a></td>
+                        {
+                        <tr>
+                            <td>@day/@month/@yearclean</td>
+                            <td>@eventss.Performance[0].Artist.DisplayName</td>
+                            <td>@eventss.Status</td>
+                            <td>@starttime</td>
+                            <td>@sun</td>
+                            <td><a href="@eventss.Uri" target="_blank">Get Tickets</a></td>
 
-                       <td></td>
-                    </tr>
+                            <td></td>
+                        </tr>
+                    }
+                }
             }
-        }
-
-
-    }
-
-
-
-
-
-    
+            }
+        </table>
         <img src="https://raw.githubusercontent.com/huberr3/IS7024/master/powered-by-songkick-pink.png" width="171" height="60">
-    
-    }
+    </div>
```

Comment:
Added missing </table> and </div> tags and moved the image URL outside of the table to align formatting with that seen in HollywoodBow.cshtml

Reply:
Good, yes... image should be outside of table.